docs/_documentations/projects-settings.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: docs
+title: Project settings 
+description: Project settings tell Codewind the specifics of your project and application 
+keywords: project, settings, Codewind, application, configuring, workspace
+duration: 1 minute
+permalink: project-settings
+type: document
+order: 4
+parent: developing-projects
+---
+
+# Project settings
+
+Project settings tell Codewind more about the specifics of your project and can affect the status or behavior of your application. Project settings can be configured from the `Project Overview` page that is accessible from a project's context menu. You can also find the project settings in the `.cw-settings` file of the project which you can edit from the IDE. The workspace automatically picks up changes to these fields.
+
+Supported project settings include:
+* [Context root](#context-root)
+* [Health check endpoint](#health-check-endpoint)
+* [HTTPS application](#https-application)
+* [Internal application port](#internal-application-port)
+* [Internal debug port](#internal-debug-port)
+* [Maven profiles](#maven-profiles)
+* [Maven properties](#maven-properties)
+* [Paths to ignore for file changes](#paths-to-ignore-for-file-changes)
+* [Project status ping timeout](#project-status-ping-timeout)
+
+## **Context root**
+`contextRoot: <string>`
+- Codewind uses the value to determine the project state.
+- When the Open Application action is used, it uses the value as the initial endpoint. 
+- If an incorrect context root is set, the project is stuck in the starting state.
+- If the health check endpoint is set, the context root is not used to determine the project state.
+- If the value is not set, the default value is `/`.
+
+## **Health check endpoint**
+`healthCheck: <string>`
+- Codewind uses the value to determine the project state.
+- Codewind expects to use this value for the application health check endpoint.
+- If a wrong health check endpoint is set, the project is stuck in the starting state.
+- If the health check endpoint is set, the context root is not used to determine the project state. 
```

Comment:
Can we also mention the default value here? I think the default value is `/` (correct me if I am wrong)